be/src/exec/olap_common.h

```diff
@@ -476,6 +518,18 @@ Status ColumnValueRange<primitive_type>::add_fixed_value(const CppType& value) {
     return Status::OK();
 }
 
+template <PrimitiveType primitive_type>
+Status ColumnValueRange<primitive_type>::add_compound_value(SQLFilterOp op, CppType value) {
```

Comment:
place compound related code together for convinient

Reply:
will change it